### ISSUE TITLE
Spousal chart gets crowded when earners have large age differences

### ISSUE THREAD

**Issue by Gregable** [Gregable/social-security-tools#63]
![82kbrkraluo](https://user-images.githubusercontent.com/25519/41820613-6dd76a74-7789-11e8-98bc-eb5b656844e9.png)

This example is the Test 1 case (1950 birthdate) with a spouse born in 1990. I assume this is uncommon, so I consider fixing this low priority.

My thought would be to actually just hide most of the years between the sliders, for example showing 2030 ... 2050 and compressing the 20 years in between them.

**Comment by stkeros** [comment]
I agree with that suggestion, to not display any lines from Age 70 of self to age 62 of spouse or vice versa. The benefit does not change for anyone during those times so one does not need the lines for any kind of reference.

As a matter of fact, why not do that for ALL cases where the filing sliders don't overlap?

**Comment by stkeros** [comment]
Two other crowding sizing issues. 1) The ages 62-70 above the sliders don't have enough room and 2) and this is trivial, the "stubs" on the lines that the slider tracks on is the same size, even though the overall slider is not as long.

**Comment by Gregable** [comment]
This is still an issue. Can reproduce it here: https://ssa.tools/calculator#pia1=1000&pia2=1000&dob1=1950-02-02&dob2=1990-02-02
